src/components/Property.astro

```diff
@@ -12,7 +14,7 @@ const isLink = customDataTypes.includes(formattedType);
       Name
     </dt>
     <dd>
-      <code>{ name }</code>
+      <Badge text={name}/>
```

Comment:
Should this be on the type? https://www.figma.com/file/IgRlQWJTyGoZhlF4fVKD6B/Centrapay-Docs-(Tailwind)?type=design&node-id=1718-454&mode=design&t=ZPbJbXOw7WVXAitK-4

Reply:
Yea I have changed this